style.css

```diff
@@ -951,226 +868,159 @@ input {
     margin-bottom: 0;
   }
 
-  .main__product {
+  .product {
     margin-bottom: 70px;
   }
 
-  .main__product_item:nth-child(n+2) {
+  .product__content:nth-child(n+2) {
     display: none;
     max-width: none;
   }
 
-  .main__product_line {
+  .product__line {
     display: none;
     max-width: none;
   }
-  
-  .main__product_item {
+
+  .product__content {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".product__content" to come before selector ".product__content:nth-child(n+2)" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484013)